packages/mdc-textfield/helper-text/mdc-text-field-helper-text.scss

```diff
@@ -0,0 +1,49 @@
+//
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+@import "../functions";
```

Comment:
do you actually need functions? I think you need variables, but I don't see a function on this page

Reply:
mdc-text-field-transition is in _functions.scss